SCF-Session-1/Cart.java

```diff
@@ -0,0 +1,85 @@
+package shoppingCart;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Cart 
+{
+    ArrayList<Product> cartItems = new ArrayList<Product>();
+    
+    public void addProduct(int productId, String name, double price, int quantity)
+    {
+    	int updateList = 0, newQuantity;
+    	Product item;
+		for(Product product : cartItems)
+		{
+			if(product.getName().equals(name))
+			{
+				updateList = 1;
+				newQuantity = product.getQuantity();
+				product.setQuantity(quantity + newQuantity);
+				break;
+			}
+		}
+		if(updateList == 0)
+		{
+			item = new Product(productId, name, price, quantity);
+			cartItems.add(item);
+		}       
+    }
+
+    public void removeProduct(int productId) 
+    {
+    	for(Product product : cartItems)
+    	{
+    		if(productId == product.getProductId())
+			{
+				cartItems.remove(product);
+				break;
+			}
+    	}
+    }
+
+    public double generateBill()
+	{
+		double sum = 0;
+		System.out.println("productId\t\tname\t\tprice\t\tquantity\ttotal price");
+		for(Product product : cartItems)
+		{
+			product.display();
+			sum += product.totalPrice();
+		}
+		System.out.println();
```

Comment:
Why empty `System.out.println();`? You can use new line character `\n` in next line to add one more line before.